src/sentry/api/endpoints/organization_details.py

```diff
@@ -81,6 +83,14 @@ class OrganizationSerializer(serializers.Serializer):
 
     def validate_slug(self, attrs, source):
         value = attrs[source]
+        if len(value) < 3:
```

Comment:
@dcramer do you have an opinion on this? Or should we not enforce a min length? For some reason I had in my head that we already did this and seemed like a good idea, but we already have a good number of people occupying the 1 and 2 character org slug space. So I'm on the fence about enforcing it now. As of today, we only require a minimum of 1 character simply due to the slug's regex serializer.

Reply:
yeah i thought we did this too -- maybe it was done in the old server side view? theres no great reason people should have 1-2 character slugs, BUT if we enforce this we probably need to consider how we resolve that situation.

Reply:
Yeah, so we have about 450 that are in the 1-2 character space today.

I assumed we'd just leave them as-is, and just enforce anything on change, which is what happens here. Not sure it's worth going through and trying to rename org slugs since we'd have to reach out to them to let them know, etc. It'll break all links and whatnot.

Reply:
Probably fine for now. May want to have a comment in here saying "Legacy orgs blah blah blah"